medcat/config_meta_cat.py

```diff
@@ -70,13 +77,15 @@ class Config:
 
 class Train(MixingConfig, BaseModel):
     """The train part of the metaCAT config"""
-    batch_size: int = 100
-    nepochs: int = 50
+    batch_size: int = 32
```

Comment:
Changing the default here could change existing behaviour. We want to avoid that.